### PR TITLE
feat: book page

### PR DESCRIPTION
**Pull request by DanericNetwork**

### CODE REVIEW

File: lib/bookPage.dart [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import 'package:biblio_app/bottomAppBar.dart';
+import 'package:flutter/material.dart';
+
+// Create a new screen
```
Comment (Chrinsieboy) [inline lib/bookPage.dart]: comment?

File: lib/bottomAppBar.dart [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                    border: InputBorder.none,
+                    hintText: 'Search Book',
+                    contentPadding: EdgeInsets.all(10.0),
+                  ),
+                ),
+              ),
+            ),
+          ),
+          IconButton(
+            icon: const Icon(Icons.qr_code_2), // replace with your desired icon
```
Comment (Chrinsieboy) [inline lib/bottomAppBar.dart]: comment?

File: lib/bottomAppBar.dart [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        children: <Widget>[
+          IconButton(
+            icon: const Icon(Icons.home_outlined), // replace with your desired icon
+            color: Colors.grey[300],
+            style: ButtonStyle(
+              // this backgroundColor makes it usable on any background color, here set to transparent
+              iconSize: MaterialStateProperty.all(40.0),
+            ),
+            onPressed: () {
+              // handle your button tap here
```
Comment (Chrinsieboy) [inline lib/bottomAppBar.dart]: comment?

File: lib/bottomAppBar.dart [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  @override
+  Widget build(BuildContext context) {
+    return BottomAppBar(
+      color: const Color.fromRGBO(30,36,56, 1),
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+        mainAxisSize: MainAxisSize.max,
+        children: <Widget>[
+          IconButton(
+            icon: const Icon(Icons.home_outlined), // replace with your desired icon
```
Comment (Chrinsieboy) [inline lib/bottomAppBar.dart]: comment?

**Review by Chrinsieboy — CHANGES_REQUESTED** [review]

**Review by Chrinsieboy — APPROVED** [review]
LGTM

File: lib/main.dart [hunk 5]
```diff
@@ -81,6 +83,7 @@ class _MyHomePageState extends State<MyHomePage> {
         // TRY THIS: Try changing the color here to a specific color (to
```
Comment (Timtendo12) [inline lib/main.dart]: Is deze comment nodig?

**Review by Timtendo12 — CHANGES_REQUESTED** [review]

**Review by Timtendo12 — APPROVED** [review]

**Pull request merged**